pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "django-outbox-pattern"
-version = "0.15.0"
+version = "0.16.0"
```

Comment:
**issue**: Version `0.15.0` uses `DEFAULT_EXCLUSIVE_QUEUE` environment variable. You changed it to something else. It's a break change. So the version should be `1.0.0`.

Reply:
This change was made from version 0.14.0 to 0.15.0, but we didn't see it as a breaking change because it doesn't affect compatibility. It works with RabbitMQ newest version (working in DEV/QA) and the old one (working in PRD).

Reply:
I get it now! Thanks for explaining 😄